Sources/TSCUtility/Platform.swift

```diff
@@ -41,7 +41,8 @@ public enum Platform {
             }
             if localFileSystem.isFile(AbsolutePath("/etc/redhat-release")) ||
                localFileSystem.isFile(AbsolutePath("/etc/centos-release")) ||
-               localFileSystem.isFile(AbsolutePath("/etc/fedora-release")) {
+               localFileSystem.isFile(AbsolutePath("/etc/fedora-release")) ||
+               Platform.isAmazonLinux2 {
```

Comment:
Given the increasing complexity here, do you mind adding a unit test? I suggest making findCurrentPlatform public and renaming it to _findCurrentPlatform. Then you can add a parameter `fs: FileSystem` with default value of localFileSystem. This should allow you to use InMemoryFileSystem in unit tests.